data_model/src/asset.rs

```diff
@@ -276,6 +263,17 @@ pub mod model {
     }
 }
 
+/// Error occurred while parsing `AssetValueType`
+#[derive(Debug, displaydoc::Display, Clone)]
+#[cfg_attr(feature = "std", derive(thiserror::Error))]
+#[repr(u8)]
+pub enum AssetValueTypeParseError {
+    /// `AssetValueType` should be either `Store` or `Numeric`
+    WrongVariant,
```

Comment:
we also have `TypeError::AssetValueType` in `isi.rs` do these overlap?

Reply:
No, this error is for parsing (`FromStr` error), while `TypeError::AssetValueType` is for mismatch in isi.